### PR TITLE
Fix undefined name.  Add animation tests.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Summary

xref https://github.com/matplotlib/matplotlib/pull/10793#commitcomment-28114249
Sorry :)

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is PEP 8 compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

**Comment by tacaswell** [comment]
Yeah, we do not force it to try to save a too-big animation to js.

**Comment by dopplershift** [comment]
Oops. 🐑

**Comment by jklymak** [comment]
... I told @anntzer that I trusted him...  and the tests.

**Comment by dopplershift** [comment]
Yeah, I see now that I didn't screw it up originally. 😁

**Comment by anntzer** [comment]
Well, I'm writing tests for the whole thing right now and it looks like (tbc?) I'm finding an unrelated bug in jshtml output... so perhaps that was a good thing :p

**Comment by anntzer** [comment]
Added some tests, including an xfailing one :-)

**Comment by tacaswell** [comment]
Issue is we don't have ffmpeg installed on appveyor.

**Comment by anntzer** [comment]
yup, working on that
stupid avconv :/

**Comment by anntzer** [comment]
Added ppa to travis to install ffmpeg (not available by default on trusty) instead of avconv, given that that's actually our default converter...

File: doc/devel/testing.rst [hunk 1]
```diff
@@ -25,7 +25,7 @@ local FreeType build
 
 The following software is required to run the tests:
 
-  - pytest_ (>=3.1)
```
Comment (jkseppan) [inline doc/devel/testing.rst]: Shouldn't this be 3.4?

Comment (anntzer) [reply doc/devel/testing.rst]: sorry, fixed

**Comment by jklymak** [comment]
Four approvals, but needs a rebase!

**Comment by anntzer** [comment]
Trying to beat the record of the most approvals on a PR :p rebased.

**Pull request merged**